### DISCUSSION TITLE
3rd party keyboards on iOS

### DISCUSSION THREAD

**Discussion by PollockApp** [SafeExamBrowser/seb-mac#315] [Q&A] [answered]
Hi,

We are trying to serve students with dyslexia, dysorthographia, and dysphasia by accommodating them with the use of Lexibar (https://www.lexibar.ca/), a 3rd party iOS keyboard.

When using SEB, the keyboard option is not available (only native-app languages and emoji). Does the single-app restriction for iOS also apply to keyboards?

Is there any way we can activate and allow a 3rd party keyboard?

Thank you!

As seen in SEB:
![IMG_6841](https://github.com/SafeExamBrowser/seb-mac/assets/11586627/ee6d19fe-0f3a-4307-aa8d-6a533e189da7)

Expected:
![IMG_6843](https://github.com/SafeExamBrowser/seb-mac/assets/11586627/0dbccc66-1e27-49c3-b409-9d86dd2334e7)

**Comment by danschlet** [comment, marked as answer]
SEB for iOS currently explicitly blocks any third-party keyboards, as those could be used for cheating. We could add a configuration option to allow third-party keyboards, but most likely it wont be possible to control which keyboards can be allowed. 

So if there is such an option, you would need to make sure that only students requiring it would be allowed to use third-party keyboards and/or you would need to make sure that only a third-party keyboard can be used which doesn't allow students to cheat for example by using spell checking in language tests or predictive text or AI functionality (which I'm sure some keyboards integrated by now).

**Reply by PollockApp** [reply]
Hi @danschlet , thank you for taking the time to review and reply to our question. That helps.

So it is my understanding that it is **currently** not possible.

If implemented, I believe it would be feasible to only assure that students with dyslexia would get access to the special configuration.
Is there a way we can make a formal backlog request? Thank you!
